Jenkinsfile

```diff
@@ -97,6 +97,9 @@ pipeline {
                     sudo DEBIAN_FRONTEND=noninteractive add-apt-repository ppa:ethereum/ethereum
                     sudo DEBIAN_FRONTEND=noninteractive apt-get install --yes ./kevm_${VERSION}_amd64.deb
                     sudo DEBIAN_FRONTEND=noninteractive apt-get install --yes solc
+
+                    # make sure we're using package installed kevm_pyk for testing
+                    rm -rf kevm_pyk
```

Comment:
:question: Otherwise, Python picks up the module in the repo's root, right?

If so, perhaps we can fix this problem by introducing another level of hierarchy in the folder structure, e.g. `scripts/kevm_pyk`, so that the same issue does not come up during development.

Reply:
Yes, that's exactly it. Ok, I'll make that adjustment.

Reply:
Adjustment made, waiting for CI to go through to verify that packaging works.